### PR TITLE
Form

### LINKED ISSUE #11: Create a new ToDo
# User story
As a user 
I want to be able to click on a plus button, 
so I can create a new ToDo and add it to the list. 

#Description

#Material

#Done?
- [x] Create a form for a new ToDo
- [x] Create an input field for a title 
- [x] Add a submit button to save an information
- [ ] Create a plus button 
- [ ] Add a link to the new ToDo page  to the button

### PR DESCRIPTION
**Pull request by AlexRyzhkova**
Fix #11 
Add a Form component for CreateToDo page:
Add Input
Add DropDown for category
Add button
Add delete button

Link to storybook: https://marry-me-app-form-yu2trrhw4hwh.herokuapp.com/storybook/?path=/story/form--create-to-do

### CODE REVIEW

File: client/src/api/fetchToDos.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+export const getTodos = async () => {
+  const response = await fetch("/api/toDos");
+
+  if (!response.ok) {
+    throw new Error(response);
+  }
+
+  const fetchedTodos = await response.json();
+  console.log(fetchedTodos);
```
Comment (jennydely) [inline client/src/api/fetchToDos.js]: delete the console log, before you merge :)

File: client/src/components/DeleteButton.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    <ButtonContainer>
+      <img src={deleteIconSrc} alt="Dustbin" />
+      <p>To-Do löschen</p>
+    </ButtonContainer>
+  );
+};
+export default DeleteButton;
+
+const ButtonContainer = styled.button`
+  display: flex;
```
Comment (jennydely) [inline client/src/components/DeleteButton.js:16]: It's a bit strange, that a button container is a button? normally a container is something like a div or span and the buttons are in this?

File: client/src/components/List/List.js [hunk 3]
```diff
@@ -81,3 +32,7 @@ const ListContainer = styled.div`
     max-width: 50rem;
   }
 `;
+
+List.propTypes = {
+  items: PropTypes.array,
+};
```
Comment (jennydely) [inline client/src/components/List/List.js:38]: I think the proptypes should be declared at the beginning of the code.

File: client/src/components/form/Form.js [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+  }
+  div:nth-child(3) {
+    align-self: flex-end;
+  }
+`;
+
+Form.propTypes = {
+  placeholder: PropTypes.string,
+  topic: PropTypes.string,
+  category: PropTypes.string,
```
Comment (jennydely) [inline client/src/components/form/Form.js:75]: same here, thing this should be at the beginning?

File: client/src/components/form/Input.js [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+        onChange={onChange}
+        placeholder={placeholder}
+      />
+    </InputContainer>
+  );
+};
+export default Input;
+
+const InputContainer = styled.label`
+  display: grid;
```
Comment (jennydely) [inline client/src/components/form/Input.js:21]: Is the label really a container? otherwise just name it StyledInput or something like that

**Review by jennydely — COMMENTED** [review]
Looks okay, we learned to write PropTypes at the beginning at the code and that Containers are something like divs, spans etc.. otherwise it's better to understand, if you name them like StyledButton or StyledLabel.

File: client/src/components/List/List.js (hunk 3 above)
Comment (lmachens) [reply client/src/components/List/List.js:38]: @jennydely this is not important. It's more common to define them at the end of the files.
See https://reactjs.org/docs/typechecking-with-proptypes.html or popular libs like material-ui:
https://github.com/mui-org/material-ui/blob/next/packages/material-ui/src/Button/Button.js

**Pull request merged**